src/olympia/amo/tests/test_fields.py

```diff
@@ -38,6 +39,14 @@ def test_catches_invalid_url(self):
         with self.assertRaises(exceptions.ValidationError):
             assert self.field.clean(u'https://test.[com')
 
+    def test_with_site_url(self):
```

Comment:
Can you test this by using override_settings to set `SITE_URL` to a known value and then literally testing with that value in the string?